tests/Security/Voter/HandleCommandVoterTest.php

```diff
@@ -50,22 +50,62 @@ public function testVote($attribute, $subject, $decision, $mapping, $expected)
-            ['handle', null, true, [], VoterInterface::ACCESS_ABSTAIN],
-            // Testcase: abstain when not handling a command and not using the handle attribute
-            ['create', null, true, [], VoterInterface::ACCESS_ABSTAIN],
-            // Testcase: abstain when not handling a command
-            ['create', new FakeCommand, true, [], VoterInterface::ACCESS_ABSTAIN],
-            // Testcase: default is unrelated to decision manager
-            ['handle', new FakeCommand, false, [], VoterInterface::ACCESS_DENIED],
-            // Testcase: deny access if decision manager returns false
-            ['handle', new FakeCommand, false, [FakeCommand::class => ['ROLE_USER']], VoterInterface::ACCESS_DENIED],
-            // Testcase: grant access if decision manager returns true and the command is in the mapping
-            ['handle', new FakeCommand, true, [FakeCommand::class => ['ROLE_USER']], VoterInterface::ACCESS_GRANTED],
-            // Testcase: deny access if the command is not in the mapping (i.e. a default deny access case)
-            ['handle', new FakeCommand, false, ['someOtherCommand' => ['ROLE_USER']], VoterInterface::ACCESS_DENIED],
+            'default access is false' => [
+                'handle',
+                new FakeCommand,
+                true,
+                [],
+                VoterInterface::ACCESS_DENIED
+            ],
+            'default is unrelated to decision manager' => [
+                'handle',
+                new FakeCommand,
+                false,
+                [],
+                VoterInterface::ACCESS_DENIED
+            ],
+            'abstain when not handling a command, but using the handle attribute' => [
+                'handle',
+                null,
+                true,
+                [],
+                VoterInterface::ACCESS_ABSTAIN
+            ],
+            'abstain when not handling a command and not using the handle attribute' => [
+                'create',
+                null,
+                true,
+                [],
+                VoterInterface::ACCESS_ABSTAIN
+            ],
+            'abstain when handling a command and not using the handle attribute' => [
+                'create',
+                new FakeCommand,
+                true,
+                [],
+                VoterInterface::ACCESS_ABSTAIN
+            ],
+            'deny access if decision manager returns false, even if the command is in the mapping' => [
+                'handle',
+                new FakeCommand,
+                false,
+                [FakeCommand::class => ['ROLE_USER']],
+                VoterInterface::ACCESS_DENIED
+            ],
+            'grant access if decision manager returns true and the command is in the mapping' => [
+                'handle',
+                new FakeCommand,
+                true,
+                [FakeCommand::class => ['ROLE_USER']],
```

Comment:
There's another, different issue with this test. Although it's near identical to the previous one, removing the mapping here for an empty array should still pass (remember, the decision manager is mocked). However, emptying the array crashes the test with the following:

```
1) League\Tactician\Bundle\Tests\Security\Voter\HandleCommandVoterTest::testVote with data set "grant access if decision manager returns true and the command is in the mapping" ('handle', League\Tactician\Bundle\Tests\Fake\FakeCommand Object (), true, array(), 1)
PHPUnit\Framework\Exception: PHP Fatal error:  Uncaught BadMethodCallException: Method Mockery_1_Symfony_Component_Security_Core_Authentication_Token_TokenInterface::serialize() does not exist on this mock object in [...]/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php(16) : eval()'d code:709
Stack trace:
#0 [...]/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php(16) : eval()'d code(839): Mockery_1_Symfony_Component_Security_Core_Authentication_Token_TokenInterface->_mockery_handleMethodCall('serialize', Array)
#1 [internal function]: Mockery_1_Symfony_Component_Security_Core_Authentication_Token_TokenInterface->serialize()
#2 -(270): serialize(Array)
#3 -(386): __phpunit_run_isolated_test()
#4 {main}
```

This appears to be related to the processIsolation flag being flipped on everywhere and, indeed, reverting the phpunit.xml.dist flag back allows this test to pass.

So we might want to switch back to making processIsolation opt-in and again, revisit this test setup.